testing/sddm/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
+pkgname=sddm
+pkgver=0.18.1
+pkgrel=0
+pkgdesc="Simple Desktop Display Manager"
+url="https://github.com/sddm/sddm/"
+arch="all"
+license="GPL-2.0-only"
+depends="elogind xorg-server dbus-x11"
+makedepends="cmake extra-cmake-modules qt5-qtbase-dev libxcb-dev upower-dev elogind-dev linux-pam-dev qt5-qtdeclarative-dev qt5-qttools-dev utmps-dev"
+subpackages="$pkgname-lang $pkgname-openrc"
+install="sddm.pre-install"
+langdir="/usr/share/sddm/translations"
+pkgusers="sddm"
+pkggroups="sddm"
+source="$pkgname-$pkgver.tar.gz::https://github.com/sddm/$pkgname/archive/v$pkgver.tar.gz
+	pam-path-fix.patch
+	sddm.openrc"
```

Comment:
please put the ending doublequote on the next line, it will end up in  better diffs in the future